types/rclone.js/index.d.ts

```diff
@@ -0,0 +1,103 @@
+// Type definitions for rclone.js 0.6
+// Project: rclone.js
+// Definitions by: atifcppprogrammer <https://github.com/atifcppprogrammer>, sntran <https://github.com/sntran>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+import type { ChildProcess } from 'node:child_process';
```

Comment:
```suggestion

import type { ChildProcess } from 'node:child_process';
```
just for readability

Reply:
Addressed in latest commit